### ISSUE TITLE
[Due for payment 2026-01-30] Screen Reader: Create Workspace: The buttons of members are announced without details

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77453]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe to focus the Workspaces tab and double-tap to activate it
3. Swipe to focus any workspace and double-tap to activate it
4. Swipe to focus the Members menu item and double-tap to activate it
5. Swipe to focus any member and note how it is announced

## Expected Result:
Each member button should announce all its displayed details, e.g.:
- Name, email address, role (if applicable)
Example:
- 'lol, [applausetester+en@applause.expensifail.com,](mailto:applausetester+en@applause.expensifail.com,) Owner'

## Actual Result:
The buttons of members are announced without details, only the member’s name is announced.

## Area issue was found in:
Create Workspace

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Combine all relevant text into the accessibilityLabel of the member button to ensure it is being announced, e.g.:
- Name, email address, role (if applicable)
Example:
- 'lol, [applausetester+en@applause.expensifail.com,](mailto:applausetester+en@applause.expensifail.com,) Owner'
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="1179" height="2437" alt="Image" src="https://github.com/user-attachments/assets/be8e2b0b-0b56-4f10-8702-e18d3caf3d61" />

https://github.com/user-attachments/assets/1652ae09-a853-4705-b6c5-ca9b81b6fc9f

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7007072
**Issue reported by:** NA
**Slack conversation:** NA
